src/BasicRum/Beacon/Importer/Process/Beacon.php

```diff
@@ -36,6 +41,18 @@ public function extract(array $beacons)
 
             $beacons[$key] = json_decode($beacon[1], true);
 
+            if ( isset($beacons[$key]['restiming']) && $beacons[$key]['restiming'] )
+            {
+                $decompressor = new ResourceTimingDecompressor_v_0_3_4();
```

Comment:
Please do not create the ResourceTimingDecompressor_v_0_3_4 in the loop. Better you create it once in the constructor and reuse it.